include/dlaf/eigensolver/tridiag_solver/merge.h

```diff
@@ -264,15 +263,10 @@ void sortIndex(SizeType i_begin, SizeType i_end, pika::shared_future<SizeType> k
                              ex::when_all_vector(tc.readwrite<SizeType, D>(out_index)));
 
   ex::start_detached(
-      di::transform<di::TransformDispatchType::Plain, false>(di::Policy<DefaultBackend_v<D>>(),
-                                                             std::move(sort_fn), std::move(sender)));
+      di::transform(di::Policy<DefaultBackend_v<D>>(), std::move(sort_fn), std::move(sender)));
 }
 
 // Applies `index` to `in` to get `out`
-//
-// Note: `pika::unwrapping()` can't be used on this function because std::vector<Tile<const >> is
-// copied internally which requires that Tile<const > is copiable which it isn't. As a consequence the
-// API can't be simplified unless const is dropped.
```

Comment:
@teonnik do you remember if there's some simplification we can do here right away now that the unwrapping has been changed? Or is "the API can't be simplified" just referring to the `transform<..., false>` call?